packages/dynamite/dynamite_end_to_end_test/lib/nested_ofs.openapi.dart

```diff
@@ -33,7 +34,8 @@ class Client extends DynamiteClient {
 
 @BuiltValue(instantiable: false)
 abstract interface class BaseInterface {
-  String? get attribute;
+  JsonObject? get required;
```

Comment:
Here in the code it adds the new property and I don't see a reason why this would be needed for the test